tensorboard/plugins/profile/README.md

```diff
@@ -1,135 +1,89 @@
+
+    - The step time averaged over all steps sampled
+    - The percentage of Host time that is idle
+    - The percentage of TPU time that is idle
+    - The percentage utilization of the TPU matrix units
+
+  * Step-time Graph, which plots a graph of step time (in milliseconds) over
+  all the steps sampled.
+
+  * Top 10 TensorFlow operations on TPU, which shows the TensorFlow operations
+  executed on the TPU that consumes the majority of time.
+
+  * Run Environment, which includes:
+
+    - The number of hosts used
+    - The type of TPU used
+    - The number of TPU cores
+    - The training batch size
+    - Job information (both the build command and run command)
+    - Recommendation for Next Steps, which first reports if the workload is
+    input bounded. Then it suggests the next tools for reducing the bottleneck
+    (depending on whether it is the input time, the TPU time, or both).
+
+## Trace Viewer
+  Trace Viewer contains a timeline that shows various operations in your
+  TensorFlow model that the TPUs and host machine executed over time.
+
+  ![Trace Viewer](docs/trace-viewer.png)
+
+  The Timeline pane contains the following elements:
+
+  * A top bar, which contains various auxiliary controls.
+  * A time axis, which shows time relative to the beginning of the trace.
+  * Section and track labels. Each section contains multiple tracks and
+  has a triangle on the left that you can click to expand and collapse the
+  section. There is one section for every processing element in the system.
+  Sections and tracks will be explained in more detail below.
+  * A tool selector, which contains various tools for interacting with the
+  Trace Viewer. Events. These show the time during which an operation was
+  executed or the duration of meta-events, such as training steps.
+
+## Op Profile
+  Op Profile tool displays the performance statistics of
+  [XLA](https://www.tensorflow.org/performance/xla) operations executed during
+  the profiling period. Op Profile shows:
+
+  * How your application uses the TPU. The TPU FLOPS utilization reported is
+    defined as the measured number of floating point operations per second
+    (FLOPS) over the peak FLOPS supported by the TPU.
+  * The most time consuming operation.
+  * Details of each op, including shape, XLA expression and padding.
+
+  ![Op Profile](docs/op-profile.png)
+
+## Input Pipeline Analyzer
+  Input pipeline analyzer tries to answer two questions:
+
+  * Is your model input bound?
+  * If it is, why?
+
+  ![Input Pipeline](docs/input-pipeline.png)
+
+  The analysis contains three sections:
+  * Summary, which tells you the overall input pipeline analysis: whether
+  your application is input bound and by how much.
+  * Device-side analysis, which shows you the detailed device-side analysis
+  results, including the device step time and how much is spent waiting for
+  the input data.
+  * Host-side analysis, which shows you the detailed analysis on the host side,
```

Comment:
I don't see this section in the screenshot - is is just below the bottom of the viewable page?  If possible, might be nice to change the window size so that it does show up in the screenshot.

Reply:
Updated the screenshot.